cm_predict.py

```diff
@@ -234,7 +234,13 @@ def mosaic(self):
         2) Sets final image size from col*width, rows*height
         3) Creates final image from all sub-tiles, and bounding box parameters are also set 
         """
-        new_im = image_grid_overlap(image_list, rows=23, cols=23, crop=16)
+        overlap_pix = self.overlapping*self.tile_size
+        if (overlap_pix % 2) != 0:
+            raise Exception ('Even number of pixels needed')
```

Comment:
Math looks fine to me, though I think this condition would be better to have in the very beginning, otherwise sub-tiling and prediction could be run with an incompatible for mosaic tile size, which basically means an useless run.
@anitera

Reply:
yes, actually makes sense. I didn't think about it

Reply:
Agree, let's move it to main function before sub-tiling call